cfpb/cfgov-refresh
# Change Go Live Date to a DateTime

The `live_stream_date` should be a DateTime.

## Changes

- Change `live_stream_date` from Date to a DateTime

## Testing

- Run migrations
- Create or modify an Event page and confirm the Go Live Date field is a DateTime

## Review

- @kave @kurtw 

## Screenshots


## Notes

-

## Todos

-

## Checklist
